app/controllers/subjects_controller.rb

```diff
@@ -15,7 +15,9 @@ def show
   def all
     subjects =
       if params[:search].present?
-        Subject.where("lower(unaccent(name)) LIKE lower(unaccent(?))", "%#{params[:search].strip}%")
+        Subject
+          .where("lower(unaccent(name)) LIKE lower(unaccent(?))", "%#{params[:search].strip}%")
+          .or(Subject.where("lower(unaccent(short_name)) LIKE lower(unaccent(?))", "%#{params[:search].strip}%"))
```

Comment:
@RenzoMinelli's idea: would it be possible to search by `code` as well?

Reply:
yep that would help a lot

Reply:
I thought about it, but wanted to do it as part of a different PR 🙂 

Having said that, I don't know how useful it will be given that no one knows the subjects codes. Anyway given the complexity of introducing that feature I don't think it hurts to add it...

Reply:
It's a different feature – and therefore requires tests – that's why I was thinking about a different PR 😬 

> some subjects have the same name but different code.

That's a very good point 👀

Reply:
>It's a different feature – and therefore requires tests – that's why I was thinking about a different PR 😬

Makes sense